extensions/wikia/AdEngine/AdEngine2ContextService.class.php

```diff
@@ -71,6 +71,7 @@ public function getContext( Title $title, $skinName ) {
 				'forceProviders' => $this->filterOutEmptyItems( [
 					'directGpt' => $wg->AdDriverForceDirectGptAd,
 					'liftium' => $wg->AdDriverForceLiftiumAd,
+					'turtle' => !!$wg->AdDriverForceTurtleAd,
```

Comment:
Why are we forcing it to be `Boolean` and other (above) aren't casted? Shouldn't we cast all of them just in matter of consistency?